app/Repositories/AppointmentRepository.php

```diff
@@ -18,6 +17,11 @@ public function __construct(Appointment $model)
        parent::__construct($model);
    }
 
+   public function filterAppointmentsByStatus()
+   {
+       return request()->user('doctor')->appointments()->filter(app(AppointmentFilters::class))->get();
```

Comment:
use `auth()` facade to access current user

Reply:
done

Reply:
done